tests/test_crawlers.py

```diff
@@ -1,17 +1,41 @@
 """Test suite for crawlers"""
-#pylint: disable=protected-access
 
 import logging
 import os
 import unittest
 import unittest.mock as mock
 from datetime import datetime, timezone
+#import geospaas_harvesting.harvest as harvest
 
 import requests
 
 import geospaas_harvesting.crawlers as crawlers
 
 
+class WebDirectoryCrawlerExceptionTestCase(unittest.TestCase):
+    """Tests for the webDirectory crawler Exceptions"""
+
+    def test_lack_of_definition_of_get_download_url(self):
+        """shall return "NotImplementedError" exception in the case of lack of definition of get_download_url method """
+        my_harvester = crawlers.WebDirectoryCrawler
+        with self.assertRaises(NotImplementedError):
+            my_harvester.get_download_url("test_text", "test_text2")
```

Comment:
Again, instantiate a harvester, don't use the class directly.
It's a very bad practice that results in extremely confusing code.

Reply:
@aperrin66 
https://github.com/nansencenter/django-geo-spaas-harvesting/pull/20#discussion_r474427475

Reply:
Done in the next commit.